.babelrc.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+const _ = require( 'lodash' );
+const path = require( 'path' );
+
+const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
+const isCalypsoServer = process.env.CALYPSO_SERVER === 'true';
+const isCalypso = isCalypsoClient || isCalypsoServer;
+
+const modules = isCalypsoClient ? false : 'commonjs'; // only calypso should keep es6 modules
```

Comment:
Tho https://nodejs.org/api/esm.html 😁

Reply:
thats exciting! do you know when it will become stable?